packages/trader/src/Modules/SmartChart/Components/all-markers.jsx

```diff
@@ -178,34 +224,81 @@ const TickContract = RawMarkerMaker(
         /** @type {CanvasRenderingContext2D} */
         const ctx = context;
 
-        const color = get_color({
+        const color = getColor({
             is_dark_theme,
             status,
             profit: is_sold ? profit : null,
         });
 
-        ctx.save();
         ctx.strokeStyle = color;
         ctx.fillStyle = color;
+        ctx.save();
 
         const draw_start_line = is_last_contract && start.visible && !is_sold;
+        const is_accumulators_contract = contract_type === 'ACCU';
         const scale = calc_scale(start.zoom);
-
         const canvas_height = canvas_fixed_height / window.devicePixelRatio;
-        if (barrier) {
-            barrier = Math.min(Math.max(barrier, 2), canvas_height - 32); // eslint-disable-line
+
+        [barrier, barrier_2].filter(Boolean).forEach(b => {
+            b = Math.min(Math.max(b, 2), canvas_height - 32); // eslint-disable-line
+        });
+
+        const entry = ticks[0];
+        const exit = ticks[ticks.length - 1];
+        const previous_tick = ticks[ticks.length - 2] || exit;
+        const opacity = is_sold ? calc_opacity(start.left, exit.left) : '';
+
+        if (start && is_accumulators_trade_without_contract) {
+            // draw 2 barriers with a shade in-between only
+            draw_partial_shade({
+                ctx,
+                start_left: start.left,
+                fill_color: 'rgba(55, 124, 252, 0.08)',
+                stroke_color: getColor({ status: 'dashed_border', is_dark_theme }),
+                top: barrier,
+                bottom: barrier_2,
+                scale,
+            });
+            return;
```

Comment:
Should we return nothing? Seems here we only need call `draw_partial_shade()`

Reply:
@NikitK-deriv affirmative. we have to stop further code execution in case it is_accumulators_trade_without_contract